sdk/python/lib/pulumi/automation/_local_workspace.py

```diff
@@ -452,6 +452,46 @@ def list_stacks(self, include_all: Optional[bool] = None) -> List[StackSummary]:
             stack_list.append(stack)
         return stack_list
 
+    def install(
+        self,
+        no_plugins: bool = False,
+        no_dependencies: bool = False,
+        reinstall: bool = False,
+        use_language_version_tools: bool = False,
+        on_output: Optional[OnOutput] = None,
+    ) -> None:
+        ver = VersionInfo(3)
+        if self.pulumi_command.version is not None:
+            ver = self.pulumi_command.version
+
+        if ver >= VersionInfo(3, 91):
```

Comment:
I'd probably have written this that the if checks were flipped then if the condition fails then raise but don't nest the rest of the method.

Reply:
yeh alright, I wouldn't fight mypy for it